src/jsMain/kotlin/org/hisp/dhis/rules/RuleEngineJs.kt

```diff
@@ -122,7 +123,7 @@ class RuleEngineJs {
         return RuleActionJs(
             data = ruleAction.data,
             type = ruleAction.type,
-            values = ruleAction.values.entries.map { e -> Entry(e.key, e.value) }.toTypedArray()
+            values = JsMap(ruleAction.values.entries.map { e -> tupleOf(e.key, e.value) }.toTypedArray())
```

Comment:
```suggestion
            values = ruleAction.values
```
Shouldn't work like this?

Reply:
They are different types. The type of `RuleAction.values` is Map, the type of `RuleActionJs.values` is JsMap.

Reply:
Of course :grimacing: